src/pages/lessons/fundamentals/eth-intro-part-2.mdx

```diff
@@ -0,0 +1,283 @@
+---
+title: A Developer's Guide to Ethereum, Pt. 2
+description: Introduction to Ethereum with web3.py and Python.
+icons: ["ethereum"]
+---
+
+import { LessonHeader } from "../../../components/mdx/LessonHeader";
+import Layout from "../../../components/Layout";
+
+<Layout
+    title="A Developer's Guide to Ethereum, Pt. 2" 
+    description="Introduction to Ethereum with web3.py and Python." 
+>
+<LessonHeader title="A Developer's Guide to Ethereum, Pt. 2" />
+
+Welcome back! In the first installment of this series we covered a lot of ground
+while interacting with a simulated Ethereum network. At this point, you should
+have at least a fuzzy idea of how to answer these questions:
+
+- What's a blockchain and what’s in a block?
+- What makes Ethereum decentralized?
+- What is ether and why is it a necessary component of the network?
+
+In this post, we’ll build on these concepts and explore some of the implications
+for developers, so return to Part 1 if you skipped it or need a refresher.
+
+## What’s next? 
+
+We’re going to take a deeper look at how you can interact with the Ethereum network, starting with accounts. There are some significant differences between Ethereum accounts and Web 2.0 accounts.
+
+<Callout emoji="💡" size="md" variant="info">
+**Note**: “Web 2.0” was coined to describe the Internet era that introduced user-generated content, e.g., social media and blogs. Ethereum and other decentralized technologies are said to be a part of the next generation of the Internet: **Web 3.0**. The abbreviation, **Web3**, is used by libraries like web3.js and web3.py, and elsewhere around the ecosystem.
+</Callout>
+
+## Web2 vs. Web3
+
+It’s hard not to collect accounts in today’s web. You have one for each social media app, news site, delivery service, retailer, and airline — to name a few. Each of these accounts lives on the company's servers, which makes you subject to their terms and conditions, privacy policy, and security practices. Your account can be frozen, deleted, censored, or altered at the discretion of the host company.
+
+Web3 represents a paradigm shift for account management: you and you alone own your Ethereum accounts. When you create an account, it’s done outside of the scope of any company and can travel with you to multiple apps. In fact, creating an Ethereum account doesn’t require interaction with the Ethereum blockchain at all. Let’s create one now to prove it.
+
+<Callout emoji="💡" size="md" variant="info">
+**Note**: This exercise is purely for educational purposes. Don't store real value in an account unless you understand the security implications. Some mistakes cannot be undone! More context to come.
+</Callout>
+
+## Creating an account
+
+Same drill as last time: the concepts will be demonstrated in an IPython shell. If you’re not a Python developer, no problem. Just follow along conceptually.
+
```

Comment:
It would be nice to continue along the theme of using a d_d_academy directory for learners' projects - also to promote good practices. It would mean a slight adjustment on the first installment of Eth Intro series

Reply:
we can revisit this, but the code is transient for the sake of conveying the concepts. there is no project to speak of, so no files to create or code to store. for now, i'll suggest we punt on this, in the interest of shipping a 'good enough' first iteration.